service/src/test/resources/core-case-data-store-api/callback-request.json

```diff
@@ -67,7 +67,7 @@
           "childInformation": "Cousin",
           "litigationIssues": "NO",
           "genderIdentification": "Unknown",
-          "representedBy" : []
+          "representedBy": []
```

Comment:
err, did I not remove this in my PR? Know @Jamesnelson117 commented. Can't remember outcome

Reply:
🤷‍♂